### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Ali Zoubir, Léonard Jouve

### DISCUSSION THREAD

**Discussion by LeonardJouve** [heig-vd-dai-course/heig-vd-dai-course#475] [Show and tell]
We would like to extend the password manager we created during the last practical work to a client / server architecture.
A client could connect to the server, authenticate and then access his passwords and / or generate new ones.

## Protocole:
LOGIN <username> <password> : Authentificate.
GET <name> <optional password> : Retrieve password entry and if a password is provided, decrypte the entry.
SET <name> <password> <optional password> : Store password entry and, if a password is provided entrypte the entry.
GENERATE <special>: Generate password, if special is set, include special characters.
LOGOUT : Disconnect.
QUIT : Disconnect and quit.

**Comment by ludelafo** [comment]
Looks good for us, thank you!

**Comment by Ali-Z0** [comment]
Commit du rendu :

https://github.com/LeonardJouve/DAI-labo-02/commit/e82a6b77e0e91f972ab053fadb0ec6e98abf87ae
e82a6b77e0e91f972ab053fadb0ec6e98abf87ae
